### PR TITLE
`dhis2` remove `Class Log`

### PR DESCRIPTION
**Pull request by mtuchi**
## Summary

remove Class Log and replaced
- `Log.success` with `console.log`
- `Log.warn` with `console.warn`
- `Log.error` with `console.error`

## Review Checklist

- [x] Does the PR do what it claims to do?
- [ ] Are there any unit tests? Should there be?
- [x] Is there a changeset associated with this PR? Should there be? Note that
      dev only changes don't need a changeset.

### CODE REVIEW

File: packages/dhis2/src/Adaptor.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ axios.interceptors.response.use(
           responseData: response.data,
         };
 
-        Log.error(newError.message);
+        console.error(`✗ Error at ${new Date()}:\n∟`, newError.message);
```
Comment (taylordowns2000) [inline packages/dhis2/src/Adaptor.js:85]: i think that printing the date is now redundant.

File: packages/dhis2/src/Adaptor.js [hunk 2]
```diff
@@ -96,7 +96,10 @@ axios.interceptors.response.use(
         // eslint-disable-next-line no-param-reassign
         response = { ...response, data: JSON.parse(response.data) };
       } catch (error) {
-        Log.warn('Non-JSON response detected, unable to parse.');
+        console.warn(
+          `⚠ Warning at ${new Date()}:\n∟`,
```
Comment (taylordowns2000) [inline packages/dhis2/src/Adaptor.js:100]: same here

File: packages/dhis2/src/Adaptor.js [hunk 3]
```diff
@@ -107,7 +110,10 @@ axios.interceptors.response.use(
 
     const details = error.response?.data;
 
-    Log.error(error.message || "That didn't work.");
+    console.error(
+      `✗ Error at ${new Date()}:\n∟`,
```
Comment (taylordowns2000) [inline packages/dhis2/src/Adaptor.js:114]: isn't the idea that the runtime prints the data already? (or decides if to print the date?)

**Review by taylordowns2000 — CHANGES_REQUESTED** [review]
need a quick guide from @josephjclark here. i thought that printing the date was not necessary since that would be optionally (and later?) handled by the runtime. is that not the case?

**Comment by josephjclark** [comment]
@taylordowns2000 Yeah, datestamps will be added to all logs by the runtime. 

That's a good catch. I'd have just removed the class wrapper but you're right, no need to include the timestamps. They're actually a bit distracting the CLI.

@mtuchi please go ahead and remove the timestamps

File: packages/dhis2/src/Adaptor.js (hunk 3 above)
Comment (josephjclark) [reply packages/dhis2/src/Adaptor.js:114]: Just to be clear the _date_ is redundant, we should just log the message please

Comment (josephjclark) [reply packages/dhis2/src/Adaptor.js:114]: Just to be clear the _date_ is redundant, we should just log the message please

File: packages/dhis2/src/Adaptor.js (hunk 1 above)
Comment (mtuchi) [reply packages/dhis2/src/Adaptor.js]: Fixed https://github.com/OpenFn/adaptors/pull/417/commits/6b89e24bb01f87c94ec229072927becfb976d56f

**Review by josephjclark — APPROVED** [review]

**Review by josephjclark — APPROVED** [review]

**Pull request merged**